test/spec/modules/quantcastBidAdapter_spec.js

```diff
@@ -71,8 +71,8 @@ describe('Quantcast adapter', () => {
 
     it('sends bid requests to Quantcast Canary Endpoint if `publisherId` is `test-publisher`', () => {
       const requests = qcSpec.buildRequests([bidRequest]);
-      const url = new URL(requests[0]['url']);
-      expect(url.hostname).to.equal(QUANTCAST_TEST_DOMAIN);
+      const hostname = requests[0]['url'].split(/[:/]/)[3];
```

Comment:
I think we should use this https://github.com/prebid/Prebid.js/blob/master/src/url.js#L27 here.